### PR TITLE
Add page type model and type

### PR DESCRIPTION
**Pull request by IKarbowiak**
* Add `PageType` model
* Add `MANAGE_PAGE_TYPES_AND_ATTRIBUTES` permission
* Add `PageType` graphql type
* Add `pageType` query
* Update `Attribute` meta permissions

Related issues: [SALEOR-1167](https://app.clickup.com/t/2549495/SALEOR-1167), [SALEOR-1350](https://app.clickup.com/t/2549495/SALEOR-1350)

# Impact

* [x] New migrations
* [x] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1167 Add PageType model and type](https://app.clickup.com/t/2549495/SALEOR-1167)

**Comment by patrys** [comment]
Task linked: [SALEOR-1350 Add models to associate attribute values with page](https://app.clickup.com/t/2549495/SALEOR-1350)

File: saleor/graphql/page/types.py [hunk 1]
```diff
@@ -28,3 +38,33 @@ class Meta:
...
         model = models.Page
+
+
+@key(fields="id")
+class PageType(CountableDjangoObjectType):
+    attributes = graphene.List(
+        Attribute, description="Page type attribute of that page type."
+    )
+    available_attributes = FilterInputConnectionField(
+        Attribute, filter=AttributeFilterInput()
```
Comment (fowczarek) [inline saleor/graphql/page/types.py:49]: Description? :police_officer:

File: saleor/page/models.py [hunk 2]
```diff
@@ -60,3 +60,20 @@ def __repr__(self):
...
+        ordering = ("slug",)
+        permissions = (
+            (
+                PageTypePermissions.MANAGE_PAGE_TYPES_AND_ATTRIBUTES.codename,
+                "Manage page types and attributes.",
+            ),
+        )
+
+    def __str__(self):
+        return self.name
```
Comment (fowczarek) [inline saleor/page/models.py:79]:
```suggestion
```

**Review by fowczarek — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**